src/component/mxgraph/StyleConfigurator.ts

```diff
@@ -163,6 +163,8 @@ export default class StyleConfigurator {
 
   private configureConditionalSequenceFlowStyle(): void {
     const style = this.cloneDefaultEdgeStyle();
+    style[this.mxConstants.STYLE_STARTARROW] = this.mxConstants.ARROW_DIAMOND_THIN;
```

Comment:
seems promising, @csouchet could you add a screenshot with this new render please?

Reply:
already done, in the PR description

![PR 257 conditional sequence flow with starting diamond](https://user-images.githubusercontent.com/4921914/81831652-e7a33d00-953d-11ea-82ab-340b52c7a604.png)